### PR TITLE
Markets dashboard update

### PR DESCRIPTION
**Pull request by vbloher**
Fix queres with rate() function, add trades count by token graph

### CODE REVIEW

**Comment by hleb-albau** [comment]
@vstavetski please, provide sceens with pr of dashboards

File: kubernetes-definitions/monitoring/dashboards/markets/Orderbooks.json [hunk 1]
```diff
@@ -69,7 +69,7 @@
         "rgba(237, 129, 40, 0.89)",
         "#d44a3a"
       ],
-      "datasource": "PROMETHEUS",
+      "datasource": "DS_PROMETHEUS",
```
Comment (hleb-albau) [inline kubernetes-definitions/monitoring/dashboards/markets/Orderbooks.json]: use PROMETHEUS, not DS_PROMETHEUS

**Review by hleb-albau — CHANGES_REQUESTED** [review]

**Comment by vbloher** [comment]
- added Trade count by token and tokens pair graphs
screen shot 2018-05-07 at 12 24 21

- width of Trades count(24h) panel was increased
screen shot 2018-05-07 at 12 28 36

File: kubernetes-definitions/monitoring/dashboards/markets/Orderbooks.json (hunk 1 above)
Comment (vbloher) [reply kubernetes-definitions/monitoring/dashboards/markets/Orderbooks.json]: fixed

**Review by hleb-albau — APPROVED** [review]

**Pull request merged**